packages/subapp-web/lib/load.js

```diff
@@ -210,6 +219,8 @@ Response: ${err || body}`
 
       const { request } = context.user;
 
+      context.user.numOfSubapps = context.user.numOfSubapps || 0;
```

Comment:
if this initializes it to `0`, wouldn't that cause line 326 below to change it to `-1`?

Reply:
Before that processSubapp() (line 348) will be called where the count will be incremented first. But still, I think its good to add an additional check before decrementing the count. I will add that.